libsycl/docs/DesignDocs/UnifiedRuntime.rst

```diff
@@ -0,0 +1,26 @@
+=====================
+Unified Runtime
+=====================
+
+.. contents::
+   :local:
+
+.. _unified runtime:
+
+Overview
+========
+
+The Unified Runtime project serves as an interface layer between the SYCL
```

Comment:
```suggestion
The Unified Runtime (UR) project serves as an interface layer between the SYCL
```

Reply:
updated in https://github.com/sergey-semenov/llvm-project/pull/2/commits/7fb24ad74ff8e0caab8509a5e53bea76399823c3